allennlp/tests/data/token_indexers/ner_tag_indexer_test.py

```diff
@@ -41,3 +41,13 @@ def test_as_array_produces_token_sequence(self):
         indexer = NerTagIndexer()
         padded_tokens = indexer.pad_token_sequence({'key': [1, 2, 3, 4, 5]}, {'key': 10}, {})
         assert padded_tokens == {'key': [1, 2, 3, 4, 5, 0, 0, 0, 0, 0]}
+
+    def test_blank_ner_tag(self):
+        tokens = self.tokenizer.split_words("allennlp is awesome.")
+        indexer = NerTagIndexer()
+        counter = defaultdict(lambda: defaultdict(int))
+        for token in tokens:
+            indexer.count_vocab_items(token, counter)
+        vocab = Vocabulary(counter)
+        # should raise no exception
+        indexer.tokens_to_indices(tokens, vocab, index_name="ner")
```

Comment:
It's not obvious in this test that the NER tags are blank.  Can you `assert` something about that in here?  Or you could just construct `Token` objects yourself that have some NER tags with values, and some with blanks.  Also, it'd be good to have an actual assert on the result of this line, that you're getting the value for "NONE" where you expect.